### DISCUSSION TITLE
Adjusting solubility for morning vs evening doses over course of multiple dose regimen

### DISCUSSION THREAD

**Discussion by choitingy** [Open-Systems-Pharmacology/Forum#982] [Questions & Problems]
I have a drug with a bioavailability that is highly influenced by the food effect. I am trying to simulate a multiple dose study in humans where the drug was administered as a single dose in the morning in a fasted state and then another dose was given in the evening but in a fed state. This bid dosing was repeated for another 6 days. 

The food effect appears to greatly affect solubility for this drug. To capture this, I am wondering if there is a way to adjust solubility for the morning vs evening over the course of the multiple dosing regimen in PK-Sim?

**Comment by StephanSchaller** [comment]
Hi @choitingy, do you have a solubility-limited absorption?

You might also play with the "volume" you administer with your oral application and food, which will define the liquid volume in which the drug dissolves. If you reduce or increase it, less/more of the drug will get dissolved?

Otherwise, a solution to switch solubility in between doses could be done in MoBi, but might need some expert MoBi knowledge.

**Comment by Christoph27** [comment]
Hi Cindy,

if you have only first order processes, you could do the following if you want to stay in PK-Sim for model development: Clone your compound and just adjust the solubility. Then you apply both compounds for fasted and fet state. Further you have to define an observer summing up the concentrations of both compounds in MoBi. This observer you can import into PK-Sim. 
Since I'm not aware of a general soultion, I just made a feature request:   
 https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1885#issue-1016468636

**Comment by choitingy** [comment]
Hi @StephanSchaller and @Christoph27! Thanks for both of your help.

Indeed, it is solubility-limited absorption. I ended up using Christoph's suggestion since it was best fit for my case - this dataset was used for evaluation and I wanted to cleanly use a previously optimized solubility value for the fed state. The only caveat was that by using this method I had to apply the meal event to both fasted and fed versions of the molecule in the simulation. However, this doesn't appear to make that big of a difference so overall it works well. Thanks for the help and neat to see a feature request resulting from it!

**Reply by Christoph27** [reply]
Hi @choitingy , glad that it helped. I don't see why it is necessary to apply a meal event also for the fasted state.

**Reply by choitingy** [reply]
@Christoph27 Right, I am not aware of a way to apply a food event in a simulation to selective molecules. So right now it is applied to both the fasted and fed state versions. Any workaround for this?

**Reply by StephanSchaller** [reply]
The food event is molecule independent. It is only time-related.

**Reply by Christoph27** [reply]
Maybe I was not clear enough with my workaround: Say you have a "drug_A". This compound you define in PK-Sim and use the solubility for the fasted state. Then you clone this drug (name e.g.: "drug_A_fed") and set the solubility for "drug_A_fed" to the value for the fed state. Then you apply "drug_A" in the morning (fasted sate) without any meal event. And you apply  "drug_A_fed" in the evening (fed state) together with a meal event. To plot the total concetration of "drug_A" plus  "drug_A_fed" you make an observer in MoBi which you can re-import into PK-Sim. Not that this works only if you have only first order processes, i.e. e.g. no Michaelis-Menten CL.

**Reply by choitingy** [reply]
I think the initial instructions were clear! I'm a bit of a rookie at MoBi so it may be a misunderstanding of how to apply the observer to simulations.

So, in PK-Sim I can make two separate simulations, one for drug_A (no meal events) and one for drug_A_fed (with the meal events), as you describe above. I can import these simulations into MoBi and make an observer that sums up the concentrations of these two molecules. When I go back to PK-Sim, I need to apply the observer to one simulation that includes both drug_A and drug_A_fed so that they can be summed up. When I do this, the respective solubilities for drug_A and drug_A_fed are intact, however, the meal events still need to be applied since the observer created in MoBi doesn't account for the concentrations of drug_A_fed with the meal effect since PVB plasma concentrations are taken from the molecules and are not influenced by any events (is this correct?). Therefore, in PK-Sim, the meal events are to be added to the single simulation for the observer to capture the effect of the food events, even if it is being applied to both the fasted and fed versions of the molecule. 

I hope that was clear! Again, it may be something about how MoBi works that is to be clarified in my brain :).

**Reply by msevestre** [reply]
@choitingy The observers just sums up the concentration. The events are not linked to any drug (A or a_fed). It is just a "thing" that modifies some parameters of your simulation (molecule-independent parameters). 
An observer is just a custom formula that does whatever you want. In your case, just the sum of A and B
Your protocol would look like so

Protocol: A   ====  A_Fed ==== A === A_fed ==== A ==== A_Fed
Event:      xxxxxxxxxx EVENT xxxxxxxxxxxxx EVENT xxxxxxxxxxx  EVENT

You only apply the event once a day in the evening

So you can do everything in PKSim except the observers that sums up the two concentration. Does it make sense?

**Reply by msevestre** [reply]
To clarify what I meant with some screenshots

![image](https://user-images.githubusercontent.com/1041237/136602657-a40ada64-2e8f-4b0f-9de0-f8c634d82392.png)

![image](https://user-images.githubusercontent.com/1041237/136602707-58e04cc5-465c-47d0-aa1d-3e3fe44c4500.png)

**Reply by choitingy** [reply]
@msevestre Thanks Michael, I had done exactly as described in the set up. However, because the food event is time-dependent, it was applied to both DrugA and DrugA_fed in the single simulation that I am also getting total concentrations with the observer. 

This is where I was stuck: "When I do this, the respective solubilities for drug_A and drug_A_fed are intact, however, the meal events still need to be applied since the observer created in MoBi doesn't account for the concentrations of drug_A_fed with the meal effect since PVB plasma concentrations are taken from the molecules and are not influenced by any events (is this correct?)."
